packages/web-components/src/card/card.styles.ts

```diff
@@ -22,6 +22,10 @@ export const CardStyles = css`
   :host(:focus-within) {
     --elevation: 8;
   }
+
+  :host * {
```

Comment:
Does this need to be on every element within the host? I've tested this simply using `:host {}` and that works perfectly. I'd prefer to not use the * selector here if we don't have to - hiding the element itself should be enough.

```suggestion
  :host {
```

Reply:
Same question from me.

Reply:
Sure, `:host {}` is also fine if we want to control elements behavior as possible as we can, I agree.